src/styles/index.scss

```diff
@@ -1,3 +1,82 @@
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100vh;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  &__minutes {
+    position: absolute;
+    animation: rotate-minutes 3600s infinite linear;
+
+    &--visible {
+      height: 20vmin;
+      width: 3vmin;
+      background-color: #0700ff;
+    }
+
+    &--invisible {
+      height: 20vmin;
+      width: 3vmin;
+    }
+
+    &--speed-up {
+      animation: rotate-minutes 600s infinite linear;
+    }
+  }
+
+  &__seconds {
+    position: absolute;
+    animation: rotate-seconds 60s infinite linear;
+
+    &--visible {
+      height: 38vmin;
+      width: 1.5vmin;
+      background-color: #2c8000;
+    }
+
+    &--invisible {
+      height: 38vmin;
+      width: 1.5vmin;
+    }
+
+    &--speed-up {
+      animation: rotate-minutes 10s infinite linear;
+    }
```

Comment:
The animation 'rotate-minutes' is used here with a duration of 10s, which seems incorrect for seconds. It should likely be 'rotate-seconds' instead, to match the intended functionality of speeding up the seconds hand.